### ISSUE TITLE
Missing/ default studio thumbnails on Staging

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1376]
**TL;DR:** Some studios with non-default thumbnails are showing up as having the default thumbnail in the Explore results page.

Staging, Firefox & Chrome on Mac OS El Capitan.

The studios seem to all be ST studios, at least at first glance?

I don't think these are missing due to https://github.com/LLK/scratch-search/issues/31 since they do appear on Production?

**Expected thumbnail**
<img width="957" alt="screen shot 2017-06-27 at 1 52 08 pm" src="https://user-images.githubusercontent.com/1320456/27602219-fe778f58-5b3f-11e7-95d2-6788ba564c7e.png">

**Staging**
<img width="953" alt="screen shot 2017-06-27 at 1 52 15 pm" src="https://user-images.githubusercontent.com/1320456/27602217-fe6d880a-5b3f-11e7-8144-78956a4b5a85.png">
<img width="1086" alt="screen shot 2017-06-27 at 1 50 06 pm" src="https://user-images.githubusercontent.com/1320456/27602218-fe741c6a-5b3f-11e7-8ba3-f7848d2fd2fc.png">

**Production**
<img width="379" alt="screen shot 2017-06-27 at 1 54 12 pm" src="https://user-images.githubusercontent.com/1320456/27602243-20e34118-5b40-11e7-89bd-d35efce14d63.png">

/cc @mewtaylor

**Comment by jwzimmer** [comment]
This seems to have to do with our having pointed Staging search/ explore to production?

**Comment by mewtaylor** [comment]
This is likely due to the fact that we're pointing our current search/explore infrastructure at production data – it's using `get_image`, which means that if the image was added after February of 2017 (when we changed our thumbnail storage system), it will fallback to the default image since it's also not going to be in our staging storage system.

Once we switch to pointing staging back at staging data, this should be resolved.

**Comment by jwzimmer** [comment]
Thanks @mewtaylor!
